content/blog/convert-a-module-for-continuous-delivery.en.md

```diff
@@ -859,7 +850,7 @@ This runs all the unit tests for the module. *It runs the tests that are*
 
 ### File `azure-pipelines.yml`
 
-1. Replace the string `Build_artefact` with `Build` (in two locations).
+1. Replace the string `Build_artefact` with `Build` (in one locations).
```

Comment:
Although I don't see much value in renaming the task, we should probably do it in the template rather than here.

Reply:
It is aesthetics - it looks prettier in Azure Pipeline with the three steps 'Build, Test, Deploy'. We should change this in the template as well so we could remove it here.